app/src/main/java/org/jellyfin/androidtv/ui/presentation/CustomRowHeaderPresenter.kt

```diff
@@ -0,0 +1,9 @@
+package org.jellyfin.androidtv.ui.presentation
+
+import androidx.leanback.widget.RowHeaderPresenter
+
+class CustomRowHeaderPresenter : RowHeaderPresenter() {
+	override fun onSelectLevelChanged(holder: ViewHolder?) {
```

Comment:
```suggestion
	override fun onSelectLevelChanged(holder: ViewHolder) {
```

I'm quite sure the parameter is not-null. Documentation doesn't explicitly state it though.
https://developer.android.com/reference/androidx/leanback/widget/RowHeaderPresenter#onSelectLevelChanged(androidx.leanback.widget.RowHeaderPresenter.ViewHolder)

Reply:
👍 okay. I just went with whatever autocomplete added lol.

Reply:
Autocomplete almost always makes it nullable so it's good to double check